### DISCUSSION TITLE
improvement in sub editor : add property for reactor automatic control limiter

### DISCUSSION THREAD

**Discussion by Slaneche** [FakeFishGames/Barotrauma#12065] [Ideas and suggestions]
in your code there is a :

TargetFissionRate = adjustValueWithoutOverShooting(TargetFissionRate, signalControlledTargetFissionRate.Value, deltaTime * **5.0f**);

I would like that **5.0f** to be configurable in sub editor for *TargetFussionRate* and an other one for *TargetTurbineOutput*

Thanks

**Comment by Regalis11** [comment]
The issue tracker is only for issue reports, not feature requests. Moving to discussions.
